lighthouse-core/gather/fetcher.js

```diff
@@ -92,6 +92,20 @@ class Fetcher {
     }
   }
 
+  // `Network.loadNetworkResource` was introduced in M88.
+  // The long timeout bug with `IO.read` was fixed in M92:
+  // https://bugs.chromium.org/p/chromium/issues/detail?id=1191757
+  // Lightrider has a bug forcing us to use the old version for now:
+  // https://docs.google.com/document/d/1V-DxgsOFMPxUuFrdGPQpyiCqSljvgNlOqXCtqDtd0b8/edit?usp=sharing&resourcekey=0-aIaIqcHFKG-0dX4MAudBEw
+  async getFetcherImplBasedOnEnv() {
+    const {milestone} = await getBrowserVersion(this.session);
+    if (milestone < 92 || global.isLightrider) {
```

Comment:
could have dropped the m92 thing but ehhh might as well continue on living until we kill the iframe fetcher.